### DISCUSSION TITLE
replace SpecFlow?

### DISCUSSION THREAD

**Discussion by temmey** [jasontaylordev/CleanArchitecture#1158] [General]
Hello 👋 

Currently in the process of starting a new project with this template. 
SpecFlow hasn't been updated in 2 years, PRs are left open for years. I'm concerned about the future of this project. 
What do you think about using other more active frameworks?
eg. https://github.com/reqnroll/Reqnroll - https://reqnroll.net/news/2024/02/from-specflow-to-reqnroll-why-and-how/

**Comment by jasontaylordev** [comment]
Thanks for bringing this up. It's a worry if a tool like SpecFlow isn't actively maintained. While I don't have a specific alternative to recommend, the one you mentioned, [Reqnroll](https://github.com/reqnroll/Reqnroll), seems like an interesting option.

I'd love to hear from others, too. If anyone has suggestions for keeping SpecFlow, or for frameworks that could replace it, please share your experiences and recommendations!
